configure.ac

```diff
@@ -1291,30 +1281,17 @@ elif test "$with_hwloc_prefix" = "embedded" ; then
    PAC_POP_FLAG([enable_visibility])
 else
    AC_CHECK_HEADERS([hwloc.h])
-   # hwloc_topology_set_pid was added in hwloc-1.0.0, which is our
-   # minimum required version
+   PAC_PUSH_FLAG([LIBS])
    AC_CHECK_LIB([hwloc],[hwloc_topology_set_pid])
+   PAC_POP_FLAG([LIBS])
    AC_MSG_CHECKING([if non-embedded hwloc works])
    if test "$ac_cv_header_hwloc_h" = "yes" -a "$ac_cv_lib_hwloc_hwloc_topology_set_pid" = "yes" ; then
       have_hwloc=yes
+      PAC_PREPEND_FLAG([-lhwloc], [WRAPPER_LIBS])
    else
       have_hwloc=no
    fi
    AC_MSG_RESULT([$have_hwloc])
-
-   # FIXME: Disable hwloc on Cygwin for now. The hwloc package,
-   # atleast as of 1.0.2, does not install correctly on Cygwin
-   AS_CASE([$host], [*-*-cygwin], [have_hwloc=no])
```

Comment:
This part is unrelated to this patch.  If we really want to not disable hwloc on cygwin, that should be a separate patch and tested.

Reply:
Dropped this part.